src/host/precomp.h

```diff
@@ -88,7 +88,7 @@ TRACELOGGING_DECLARE_PROVIDER(g_hConhostV2EventTraceProvider);
 #include "../inc/conattrs.hpp"
 
 // TODO: MSFT 9355094 Find a better way of doing this. http://osgvsowi/9355094

```

Comment:
to-done?

Reply:
The fact that it still exists... sucks. I'll kill this when I get rid of any conversion between HR->NT :)